runtime/ftplugin/elixir.vim

```diff
@@ -23,7 +23,41 @@ if exists('loaded_matchit') && !exists('b:match_words')
         \ ',{:},\[:\],(:)'
 endif
 
+setlocal shiftwidth=2 softtabstop=2 expandtab iskeyword+=!,?
+setlocal comments=:#
 setlocal commentstring=#\ %s
 
 let &cpo = s:save_cpo
 unlet s:save_cpo
+
+let &l:path =
+      \ join([
+      \   'lib/**',
+      \   'src/**',
+      \   'test/**',
+      \   'deps/**/lib/**',
+      \   'deps/**/src/**',
+      \   &g:path
+      \ ], ',')
+setlocal includeexpr=elixir#util#get_filename(v:fname)
```

Comment:
This function doesn't exist.

Reply:
Ah, it seems to refer to this:
https://github.com/elixir-editors/vim-elixir/blob/527e6fd8798638a79621e0b5c788b67b2b4b4dbc/autoload/elixir/util.vim